os/kernel/debug/mem_leak_checker.c

```diff
@@ -414,3 +404,40 @@ int run_mem_leak_checker(int checker_pid, char *bin_name)
 
 	return OK;
 }
+
+int run_all_mem_leak_checker(int checker_pid)
```

Comment:
Could you check whether the `checker_pid` is necessary or not?

Reply:
Yes, we need to skip checking the stack region of the checker, as it may contain the address that we are checking. So we need the pid to get the stack limits and exclude it during the checking.